resources/css/humhub.wiki.min.css

```diff
@@ -1 +1 @@
play:none}.wiki-page-sidebar #category_list_view.wiki-page-list{margin-left:0;margin-right:0}.wiki-page-sidebar #category_list_view.wiki-page-list .page-title:hover{border-radius:4px}.wiki-page-sidebar .wiki-page-content-header{padding:14px 8px 20px}.wiki-page-sidebar .wiki-page-content-header h3{color:var(--text-color-main);font-weight:400;text-transform:none;font-size:12px;margin-right:16px}.wiki-page-sidebar .wiki-page-content-header h3,.wiki-page-sidebar .wiki-page-content-header h3 .fa{font-size:28px}.wiki-page-sidebar .wiki-page-content-header .btn-sm{padding:4px 8px;margin-top:4px}.wiki-page-sidebar .wiki-page-content-header .btn-add-page{display:none}.wiki-page-sidebar .wiki-search-form{float:left;clear:none;max-width:none;margin:0}}.wiki-page-list-table .wiki-search-form{float:left}.wiki-page-list-table .table-responsive{padding-left:15px;padding-right:15px}.wiki-page-list-table .table-responsive .grid-view{padding:15px 0}.wiki-page-list-table .table-responsive .grid-view .pagination{margin-bottom:0}.wiki-page-list-table .table-responsive table a:hover{color:var(--info)}.wiki-page-list-table .table-responsive table .wiki-page-list-row-title{font-size:14px;font-weight:500}.wiki-page-list-table .table-responsive table .wiki-page-list-row-title a{display:block}.wiki-page-list-table .table-responsive table .wiki-page-list-row-title a:hover{color:var(--text-color-highlight)}.wiki-page-list-table .table-responsive table td{font-size:14px;border:none}.wiki-page-list-table .table-responsive table td:hover{cursor:pointer}.wiki-page-list-table .table-responsive table td:first-child{padding-left:12px;border-top-left-radius:4px;border-bottom-left-radius:4px}.wiki-page-list-table .table-responsive table td:last-child{border-top-right-radius:4px;border-bottom-right-radius:4px}.wiki-page-list-table .table-responsive .pagination>.active>a,.wiki-page-list-table .table-responsive .pagination>.active>a:focus,.wiki-page-list-table .table-responsive .pagination>.active>a:hover,.wiki-page-list-table .table-responsive .pagination>.active>span,.wiki-page-list-table .table-responsive .pagination>.active>span:focus,.wiki-page-list-table .table-responsive .pagination>.active>span:hover{background-color:var(--info);border-color:var(--info)}@media (max-width:1199px){.wiki-page-list-table .table-responsive{border:none}.wiki-right-part:not(.visible-lg){width:100%!important}}@media (max-width:768px){.layout-content-container .wiki-page-content .wiki-page-body{padding-left:0;padding-right:0}}.wiki-preview{border:1px solid #e1e1e1;border-radius:4px;padding:10px}.wiki-preview .wiki-preview-content{margin-bottom:10px}.wiki-icon-print{float:right;cursor:pointer}@media print{#topbar-first,.comment-container,.footer-nav,.social-controls,.space-nav,.wiki-content-info,.wiki-icon-print,.wiki-menu{display:none}.layout-content-container .wiki-content{border:none}.space-layout-container{margin:0}#page-content-wrapper{padding:0}}.toggle-numbering{margin-left:auto;padding-right:10px}
```

Comment:
Please don't commit minimized/compiled files.
It is true that these are contained directly in GIT, but as far as I know this is done automatically by the build system before a release. Otherwise it leads to many problems, as there will be constant conflicts.